src/main/resources/order-api.yml

```diff
@@ -0,0 +1,208 @@
+info:
+  title: LostInBerlin API
+  description: Move your app forward with the LostInBerlin API
+  version: "1.0.0"
+# the domain of the service
+host: lost-in-berlin.com
+# array of all schemes that your API supports
+schemes:
+- https
+# will be prefixed to all paths
+basePath: /api/v1
+securityDefinitions:
+  apikey:
+    type: token
+    name: server_token
+    in: header
+produces:
+- application/json
+paths:
+  /orders:
+    get:
+      summary: Existing orders
+      description: The endpoint returns information about orders assigned to this business.
+      The response includes the order details, and lists the orders according to their execution time.
+      parameters:
+      - name: order id
+        in: query
+        description: id of the current business
+        required: true
+        type: string
+      security:
+      - token: []
+      tags:
+      - orders
+      responses:
+        "200":
+          description: An array of orders
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Order'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: Adds a new order
+      description: The endpoint sends details of an order provided by the client, containing content of the following fields: name, phone number, e-mail, date, time, number of participants, additional order information and order status.
+                     The response includes the result of the operation.
+      consumes:
+      - application/json # Media type
+      produces:
+      - application/json # Media type
+      parameters:
+      - name: order
+          in: body
+          description: order entity
+          required: true
+          schema:  # Request body contents
+            $ref: '#/components/schemas/Order' # Reference to an object
+          examples:    # Child of media type
+            Lena:   # Example 1
+              value:
+                name: Lena
+                phone: +49 176 68558497
+                eMail: aoneko@gmx.de
+                date: 27.09.2018
+                time: 15:00-20:00
+                numberParticipants: 1
+                description: Hi, I want a dog sled tour in Berlin.
+      responses:
+        "200":
+          description: Result of the operation
```

Comment:
Since we create an order and repond to the client if the order is successfully created it may make sense to use 201 - created.